plottable.js

```diff
@@ -4945,12 +4903,16 @@ var Plottable;
              */
             Category.prototype._measureTicks = function (axisWidth, axisHeight, scale, ticks) {
                 var _this = this;
+                var axisSpace = this._isHorizontal() ? axisWidth : axisHeight;
+                var totalOuterPadding = 2 * scale.outerPadding();
```

Comment:
totalOuterPaddingRatio
Padding to me means a concrete measure, while this is the ratio between padding width and band width